akoshkin/03.CSS/Add_Menu.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html> 
+<HTML> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Menu_1</TITLE>
+		<style type="text/css">
+        ul {
+            margin: 0px;
+            padding: 0px;
+            list-style-type: none;
+            border-bottom: 1px solid black;
+            border-radius: .3em;
+            width: 150px;
+            }
+        li {position: relative;}
+        li ul {
+            display: block;
```

Comment:
This variant has issue (display block, even invisible, will conflict with another same sections)